scripts/enum/msg.lua

```diff
@@ -95,7 +95,7 @@ xi.msg.basic =
     MAGIC_ABSORB_MND       = 334, -- <caster> casts <spell>. <target>'s MND is drained.
     MAGIC_ABSORB_CHR       = 335, -- <caster> casts <spell>. <target>'s CHR is drained.
     MAGIC_ERASE            = 341, -- <caster> casts <spell>. <target>'s <status> effect disappears!
-    MAGIC_STEAL            = 430, -- <caster> casts <spell>. 1 of <target>'s effects is drained.
+    MAGIC_STEAL            = 430, -- <caster> casts <spell>. <number> of <target>'s effects is drained.
```

Comment:
This also needs to be added to `message_basic.h`

Reply:
added